README.md

```diff
@@ -34,6 +34,22 @@ From elsewhere:
 
 Environment variables can be used with either script to override default local source directories and remote image repositories.
 
+### Configuring Honeycomb
+To enable Honeycomb event emission, write out a `honeycomb-data.yml` file containing your
+write key and the name of your dataset
+```
+---
+honeycomb:
+  write_key: SOME_WRITE_KEY
+  dataset: my-dataset
+```
+
+Then, pass the honeycomb overlays and your `data.yml` file to the `rollout-sh`
+script or the underlying `ytt` command
+```
+-f ./overlays/honeycomb.yml -f ./overlays/honeycomb-values.yml -f ./honeycomb-data.yml
```

Comment:
I think in theory this should be possible using a library value reference

Reply:
Would it be (a tad) friendlier from a user's perspective if:
1. `honeycomb.yml` and `honeycomb-values.yml` are collected in the same directory: `./overlay/honeycomb/enable-honeycomb.yml` and `./overlay/honeycomb/values.yml`...
1. we ask them to write their file named `honeycomb-values.yml` (so as to not introduce another term "data")

Then the instruction would read:

```
-f ./overlays/honeycomb/ -f ./honeycomb-values.yml
```